demisto_sdk/tests/test_files/layout-invalid3.json

```diff
@@ -0,0 +1,44 @@
+{
+	"typeId": "ExtraHop Detection",
+	"kind": true,
+	"version": 55,
+	"layout": {
+		"id": "ExtraHop Detection",
+		"version": 55,
+		"modified": "2019-06-20T16:01:05.659574019-07:00",
+		"name": "",
+		"kind": "close",
+		"typeId": "ExtraHop Detection",
+		"system": false,
+		"sections": [
+			{
+				"id": "",
+				"version": 0,
```

Comment:
Let's change this to -1. It will still fail but this way we are checking both cases (in this case it is valid on the inside one bit not on the outside one)

Reply:
We check that in the other files